js/components/contactList/contactList_controller.js

```diff
@@ -92,10 +139,8 @@ angular.module('contactsApp')
 					}
 				});
 			}
-			// No contact previously loaded, let's load the first of the list if not in mobile mode
-			if(ctrl.loading && $(window).width() > 768) {
-				ctrl.setSelectedId(ctrl.contactList[0].uid());
-			}
+			var firstNames = ctrl.contactList.slice(0, 20).map(function (c) { return c.displayName(); });
+			ContactService.getFullContacts(firstNames);
```

Comment:
By selecting the first uid after load, we display the first contact.
In mobile, since the list view and detail views are split, we need to stay on the contact list only. :)